src/api.js

```diff
@@ -15,7 +15,21 @@ export async function getMovie(id) {
 }
 
 export default async function getData() {
-    const res = await fetch(APIData + "/movies")
+    const res = await fetch(APIData + "/movies");
```

Comment:
viktigast av allt

Reply:
Nice work!
I have the following remarks:
getMovies() and getData() is the same function with different names
getReviews() and getScreenings() are supposed to be populated with a particular movie

Reply:
The getData() is for the test in the previous assignment, for some reason the already existing getMovies() never worked for the test but getData() did. But getData() could be removed since it doesn't really serve any purpose anymore.

getReview() and getScreenings() are going to get populated later on, it's in another Trello card. This was just to get a base template of the API imports so we could all get started with the same base.

@solidtop You can remove getData() since it's only used for old tests.

Reply:
Great! Then i'll remove getData() 👍

Reply:
> Nice work! I have the following remarks: getMovies() and getData() is the same function with different names getReviews() and getScreenings() are supposed to be populated with a particular movie

Good catch! getReviews and getScreenings should be able to populate with those query strings. Should I maybe make a parameter in those functions so we can pass in ?populate=movie etc?